src/airport.js

```diff
@@ -29,16 +22,18 @@ class Airport {
       }
       this.landedPlanes.push(plane);
       return true;
+    } else if (this.Weather.weatherStatus === `stormy`) {
```

Comment:
I attempted to refactor this function without using the else if statement, similar to the flyPlane function but was unable to achieve this. In future it would be good to understand what is happening to the code when it stops working when refactoring. This will improve my knowledge of the code functionality and overall produce nicer looking code